### ISSUE TITLE
training time

### ISSUE THREAD

**Issue by Aacrobat** [csbhr/CDVD-TSP#2]
thanks for your great project and wonderful temporal prior!
When I retrain on your code (there is no change except removing the test() func, the training time is very long . I wonder if you have this problem during training?)
In addition, in order to speed up the training, I removed a part of the prequel in the optical flow network. I wonder if this part is useful?
change
` warped01, _, _, flow_mask01 = self.flow_net(frame_list[1], frame_list[0])` 
into (code/model/cdvd_tsp.py)
` warped01, flow_mask01 = self.flow_net(frame_list[1], frame_list[0])`
change
`# warp
        frame_2_grad_warp, _ = self.warp(frame_2_grad, flow.detach())
        frame_2_warp, mask = self.warp(frame_2, flow)
        return frame_2_warp, flow, frame_2_grad_warp, mask
`

into(code/model/flow_pwc.py line 95)
`# warp
        frame_2_warp, mask = self.warp(frame_2, flow)
         return frame_2_warp, mask
`

Thanks in advance！

**Comment by csbhr** [comment]
Thanks! The main reason for the long training time is cascade training. Reducing the return values of the function cannot effectively reduce the training time.
